api/src/main/java/org/openmrs/module/initializer/api/privileges/PrivilegesLoader.java

```diff
@@ -12,4 +14,8 @@
 	public void setParser(PrivilegesCsvParser parser) {
 		this.parser = parser;
 	}
+	
+	@Override
+	protected void preload(File file) {
+	}
```

Comment:
Privileges do not have a UUID, so they can't rely on this display mechanism anyway.